documentation/03-basic-crud.markdown

```diff
@@ -126,9 +90,8 @@ $authors = AuthorQuery::create()
   ->find();
 ```
 
-When you pass a value to a `filterByXXX()` method, Propel uses the column type to escape this value in PDO. This protects you from SQL injection risks.
-
->**Tip**`filterByXXX()` is the preferred method for creating queries. It is very flexible and accepts values with wildcards as well as arrays for more complex use cases. See [Column Filter Methods](/documentation/reference/model-criteria.html#column_filter_methods) for details.
+> [!TIP]
+> `filterByXXX()` is the preferred method for creating queries. It is very flexible and accepts values with wildcards as well as arrays for more complex use cases. See [Column Filter Methods](/documentation/reference/model-criteria.html#column_filter_methods) for details.
```

Comment:
I know you didn't write that, but what does "is the preferred method for creating queries" mean? That's nonsense, isn't it?
As a user, don't tell me what I prefer, you're not the boss of me. Also, preferred over what? And it doesn't actually create queries but filters.

And what does "it's flexible and accepts wildcards and arrays" mean?